astro.config.mjs

```diff
@@ -179,6 +185,10 @@ export default defineConfig({
               label: "mctl (CLI)",
               link: "/reference/cli/",
             },
+            {
+              label: "Public roadmap",
+              link: "/reference/roadmap/",
+            },
```

Comment:
We should bump this up to the top level of the nav. `Reference` is intended to be more detailed API references.